### ISSUE TITLE
[stable/jenkins] rbac install "role"

### ISSUE THREAD

**Issue by techlunacy** [helm/charts#12486]
**Is this a request for help?**: YES

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
FEATURE REQUEST maybe documentation?
what should the values of the RBAC block be when configuring for the Role kind of configuration?
eg: 
```
  rbac: 
    install: true
    roleRef: admin
    roleKind: Role
    roleBindingKind: RoleBinding
    serviceAccountName: default
```
these dont work but i dont know what they should be
any advice would be much appreciated.

**Version of Helm and Kubernetes**:
helm 2.11.0
kubectl 1.11.0
**Which chart**: [stable/jenkins]

**Comment by maorfr** [comment]
this will very soon not be required at all!
see #12790 for more information.

**Comment by torstenwalter** [comment]
Could you try the latest version of the chart. rbac is installed now by default.

**Comment by techlunacy** [comment]
resolved by #12790
